packages/block-library/src/accordion-group/index.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ * Server-side rendering of the `core/accordion-group` block.
+ *
+ * @package WordPress
+ * @since 6.6.0
+ *
+ * @param array $attributes The block attributes.
+ * @param string $content The block content.
+ *
+ * @return string Returns the updated markup.
+ */
+
+function render_block_core_accordion_group( $attributes, $content ) {
+	$suffix = wp_scripts_get_suffix();
+	if ( defined( 'IS_GUTENBERG_PLUGIN' ) && IS_GUTENBERG_PLUGIN ) {
+		$module_url = gutenberg_url( '/build/interactivity/accordionGroup.min.js' );
+	}
+
+	wp_register_script_module(
+		'@wordpress/block-library/accordion-group',
+		isset( $module_url ) ? $module_url : includes_url( "blocks/accordion-group/view{$suffix}.js" ),
+		array( '@wordpress/interactivity' ),
+		defined( 'GUTENBERG_VERSION' ) ? GUTENBERG_VERSION : get_bloginfo( 'version' )
+	);
+
+	wp_enqueue_script_module( '@wordpress/block-library/accordion-group' );
+
+	if ( ! $content ) {
+		return $content;
+	}
+
+	$p         = new WP_HTML_Tag_Processor( $content );
+	$autoclose = $attributes['autoclose'];
```

Comment:
This should probably be escaped or converted to a boolean to prevent people from injecting nasty things into the JSON.

```suggestion
	$autoclose = (bool) $attributes['autoclose'];
```